### DISCUSSION TITLE
kV RT Dose Calculation Feasibility in matRad

### DISCUSSION THREAD

**Discussion by Jeffery-JW-Shi** [e0404/matRad#691] [Q&A]
Hi matRad team,

I am interested in understanding the feasibility of implementing kV RT dose calculation within the matRad platform. Is this feature currently supported? If not, could you guide me on how I might integrate kV dosimetry into matRad?

I appreciate any advice or information you can provide on this matter.
Thank you in advance for your assistance.

Jeffery

**Comment by wahln** [comment]
Hi Jeffery,
we don't have a kV machine in matRad. Our default generic photon machine represents a 6MV machine and provides pencil-beam kernels for use with our pencil-beam algorithm.
I've seen some work to do kV calculations with matRad for small animal irradiation by creating pencil-beam kernels to use with our pencil-beam algorithm. In principle, by interfacing an appropriate Monte Carlo code, kV calculations should also be possible, but this needs to be done manually on the user side.
